### PR TITLE
Nav updates, content edits

### PR DESCRIPTION
**Pull request by claycrenshaw**
After this PR, every experimental component will have Usage content.

New side nav structure.

Grammar fixes across the board.

Edits for entire "Patterns" section (fka Utilities).

### CODE REVIEW

**Comment by alisonjoseph** [comment]
Will take a look through tomorrow, thanks @claycrenshaw

**Comment by chrisconnors-ibm** [comment]
Where can I see this branch (rather than review diffs);
Are we confident that any possible impact to outside referrers has been mitigated?

**Comment by claycrenshaw** [comment]
When we change the URL structure for Patterns (fka Utilities) we will need to set up redirects for those pages.

**Comment by chrisconnors-ibm** [comment]
Vastly prefer 'Patterns' to 'Utilities'.

**Comment by claycrenshaw** [comment]
Good catch on those labels! @alisonjoseph

**Comment by claycrenshaw** [comment]
@alisonjoseph I don't know the proper way to update that folder structure and create the redirects. . . is that something you can take care of?

File: src/data/navigation/navigation-bak.json [hunk 1]
```diff
@@ -0,0 +1,334 @@
+{
```
Comment (alisonjoseph) [inline src/data/navigation/navigation-bak.json]: do we need this file?

File: src/data/navigation/navigation.json [hunk 2]
```diff
@@ -297,7 +272,7 @@
     }
   },
   "utilities": {
```
Comment (alisonjoseph) [inline src/data/navigation/navigation.json]:
```suggestion
  "patterns": {
```

Sorry I missed this the first time, we need to upate this also to get the links to update correctly.

File: src/data/navigation/navigation.json [hunk 3]
```diff
@@ -316,22 +291,61 @@
...
+    "sub-nav": {
+      "designers": {
+      "title": "Designers"
+    },
+    "developers": {
+      "title": "Developers"
+    }
+  }
+  },
+  "about": {
```
Comment (alisonjoseph) [inline src/data/navigation/navigation.json]:
```suggestion
  "updates": {
```

Did we want this section to be updates in the nav ? Will need to update this and also update the labels for the pages inside the section. (they both say about)

File: src/data/navigation/navigation.json (hunk 2 above)
Comment (claycrenshaw) [reply src/data/navigation/navigation.json]: Fixed

File: src/data/navigation/navigation.json (hunk 3 above)
Comment (claycrenshaw) [reply src/data/navigation/navigation.json]: Yes, updated.

File: src/data/navigation/navigation.json (hunk 2 above)
Comment (claycrenshaw) [reply src/data/navigation/navigation.json]: updated

File: src/data/navigation/navigation.json (hunk 3 above)
Comment (claycrenshaw) [reply src/data/navigation/navigation.json]: updated

**Pull request merged**